plone/app/locales/locales/fr/LC_MESSAGES/atcontenttypes.po

```diff
@@ -439,7 +439,7 @@ msgstr "Restreint les résultats de recherche des collection(s) parente(s) en ut
 #. Default: "Specify whether you want to find values lesser than, greater than, equal to, or between the chosen value(s)."
 #: ATContentTypes/criteria/simpleint.py:70
 msgid "help_int_criteria_direction"
-msgstr "Indiquez si vous cherchez des valeurs plus petites, plus grandes, égales ou entre les valeur(s) choisie(s)."
+msgstr "Spécifiez si vous cherchez des valeurs plus petites, plus grandes, égales ou entre les valeur(s) choisie(s)."
```

Comment:
Note here I didn't use "Saisissez" because it's not an input text or textarea field.